packages/ui/components/icon/Icon.tsx

```diff
@@ -10,6 +12,10 @@ function Icon({
   name: IconName;
   size?: number | string;
 }) {
+  if (!process.env.NEXT_PUBLIC_WEBAPP_URL) {
+    return <ExternalFallbackIcon {...props} name={name as unknown as ExternalFallbackIconName} />;
+  }
+
   return (
     <svg height={size} width={size} {...props} aria-hidden>
       <use href={`/icons/sprite.svg#${name}`} />
```

Comment:
Since icons will be publicly hosted on app.cal.com, we should get them from there it will reduce bundle size. So if we're using icons anywhere outside of the web app the url path can be something like 

> <use href={`app.cal.com/icons/sprite.svg#${name}`} />

But this can only be done after app.cal.com has icons which is rolled back for the time being

cc: @ThyMinimalDev